scaffolding-ruby/lib/scaffolding.sh

```diff
@@ -393,8 +393,9 @@ scaffolding_run_assets_precompile() {
 
   if _has_gem rake && _has_rakefile; then
     pushd "$scaffolding_app_prefix" > /dev/null
-    if _rake -P --trace | grep -q '^rake assets:precompile$'; then
```

Comment:
if you were trying to make this work I believe a simple `grep -Eq` would work or more specifically `grep --extended-regexp --quiet`